src/main/java/com/work/management/entity/Project.java

```diff
@@ -0,0 +1,50 @@
+package com.work.management.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * This class represents a single row in Projects Table.
+ */
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "PROJECTS")
+public final class Project {
+
+    @Column(name = "name")
+    String name;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    String id;
+
+    @Column(name = "owner")
+    Integer owner;
+
+    @Column(name = "startDate")
+    Date startDate;
+
+    @Column(name = "endDate")
+    Date endDate;
+
+    @Column(name = "createdAtTimeStamp")
+    Date createdAtTimeStamp;
+
+    @Column(name = "lastUpdatedTimeStamp")
+    Date lastUpdatedTimeStamp;
+
```

Comment:
Use @CreatedDate and @LastModifiedDate on createdAtTimeStamp and lastUpdatedTimeStamp respectively.

https://stackoverflow.com/questions/49954812/how-can-you-make-a-created-at-column-generate-the-creation-date-time-automatical